server/session/world.go

```diff
@@ -117,6 +117,8 @@ func (s *Session) ViewEntity(e world.Entity) {
 			s.writePacket(&packet.PlayerList{ActionType: packet.PlayerListActionRemove, Entries: []protocol.PlayerListEntry{{
 				UUID: v.UUID(),
 			}}})
+		} else {
+			s.ViewSkin(e)
```

Comment:
Is there a reason not to send the skin regardless of whether the entity is an actual player?

Reply:
The skin has already been set via PlayerList